packages/core/src/container/property-list/property-item-color.tsx

```diff
@@ -0,0 +1,90 @@
+import * as Components from '@meetalva/components';
+import * as MobxReact from 'mobx-react';
+import * as Model from '@meetalva/model';
+import * as React from 'react';
+import { ViewStore } from '../../store';
+import { debounce } from 'lodash';
+
+const OutsideClickHandler = require('react-outside-click-handler').default;
+
+export interface PropertyItemColorProps {
+	property: Model.ElementProperty;
+}
+
+export interface PropertyItemColorState {
+	propertyOverlay: boolean;
+}
+
+@MobxReact.inject('store')
+@MobxReact.observer
+export class PropertyItemColor extends React.Component<PropertyItemColorProps> {
+	public state = {
+		displayColorPicker: false,
+		color: (this.props.property.getValue() || '') as string
+	};
+
+	private commit = debounce(() => {
+		const props = this.props as PropertyItemColorProps & { store: ViewStore };
+		props.store.commit();
+	}, 300);
+
+	private handleColorPickerChange = (color: { hex: string }) => {
+		const { property } = this.props;
+		property.setValue(color.hex);
```

Comment:
We might enable the opacity controls by using a string format that supports it - `color.rgb` seems to be available

Reply:
We had problems with translating the color object from "react-color" (color picker) and "color" (color translation)